traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/models/Traveler.java

```diff
@@ -64,7 +63,8 @@ protected void main() {
         BlockTask sessionBeginBlockTask = new BlockTask() {
             @Override
             protected void main() {
-                if (!TextUtils.isEmpty(sessionBeginTask.getSession().getToken().getValue())) {
+                Token token = sessionBeginTask.getSession().getToken();
+                if (null != token && !TextUtils.isEmpty(token.getValue())) {
```

Comment:
Did you reach a null pointer exception here?

Reply:
I did, in the past. today I ran this code without the null check and it seems to work fine- I will remove this change